### PR TITLE
Srivathsa

### PR DESCRIPTION
**Pull request by srivathsa-002**

### CODE REVIEW

File: ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/controllers/AuthController.java [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+	LoginDTO tempLogin = new LoginDTO();
+	
+	@GetMapping("/login")
+	public String loginPage() {
+		return "login-page";
+	}
+	
+	@PostMapping("/validate")
+	public String validate(@ModelAttribute("login")LoginDTO login,Model model) {
+		tempLogin.setUsername("srivathsa");
```
Comment (ghost) [inline ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/controllers/AuthController.java]: I didn't understand this part. Where are you using the actual login object?

File: ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/dto/LoginDTO.java [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+	public LoginDTO() {
+		
+	}
+	
+	public LoginDTO(String username,String password) {
+		this.username = username;
+		this.password = password;
+	}
+	
+	public String getUsername() {
```
Comment (ghost) [inline ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/dto/LoginDTO.java]: no lombok yet

**Review by ghost — CHANGES_REQUESTED** [review]

File: ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/dto/LoginDTO.java (hunk 2 above)
Comment (srivathsa-002) [reply ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/dto/LoginDTO.java]: Yeah there was a problem with eclipse but now I switched to IntelliJ and updated it to Lombok

File: ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/controllers/AuthController.java (hunk 1 above)
Comment (srivathsa-002) [reply ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/controllers/AuthController.java]: Updated!

Comment (srivathsa-002) [reply ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/controllers/AuthController.java]: Changed

**Review by Madhuri1518 — APPROVED** [review]

**Pull request closed without merging**